ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/snmp-settings.j2

```diff
@@ -102,18 +117,27 @@ snmp-server location {{ snmp_server.location }}
 {%             if user.group is arista.avd.defined %}
 {%                 set user_cli = user_cli ~ " " ~ user.group %}
 {%             endif %}
+{%             if user.remote_address is arista.avd.defined %}
+{%                 set user_cli = user_cli ~ " remote " ~ user.remote_address %}
+{%                 if user.udp_port is arista.avd.defined %}
+{%                     set user_cli = user_cli ~ " udp-port " ~ user.udp_port %}
+{%                 endif %}
+{%             endif %}
 {%             if user.version is arista.avd.defined %}
 {%                 set user_cli = user_cli ~ " " ~ user.version %}
 {%             endif %}
 {%             if user.auth is arista.avd.defined
                   and user.version is arista.avd.defined('v3')
                   and user.auth_passphrase is arista.avd.defined %}
+{%                 if user.localized is arista.avd.defined %}
+{%                     set user_cli = user_cli ~ " localized " ~ user.localized %}
+{%                 endif %}
 {%                 set user_cli = user_cli ~ " auth " ~ user.auth ~ " " ~ user.auth_passphrase %}
-{%             endif %}
-{%             if user.priv is arista.avd.defined
-                  and user.version is arista.avd.defined('v3')
-                  and user.priv_passphrase is arista.avd.defined %}
-{%                 set user_cli = user_cli ~ " priv " ~ user.priv ~ " " ~ user.priv_passphrase %}
+{%                 if user.priv is arista.avd.defined
+                      and user.version is arista.avd.defined('v3')
```

Comment:
this check for v3 is redundant since it is nested under the `auth` which also checks for this.

Reply:
Missed this one - addressed in 81d0d63 as well